README.md

```diff
@@ -2,13 +2,18 @@
 Secure banking User interfaces
 
 ## Modules
-### Secure Banking cli readme
+### Secure Banking cli UI
 [securebanking-cli-ui readme](securebanking-cli-ui/README.md)
 
-### Secure Banking common readme
+### Secure Banking common UI
 [securebanking-common-ui readme](securebanking-common-ui/README.md)
 
-### Secure Banking RCS
+### Secure Banking RCS UI
 - Bank simulation (Resource Consent Service)
 
 [securebanking-rcs-ui readme](securebanking-rcs-ui/README.md)
+
+### Secure Banking Auth UI
+- OIDC Login/Register pages
+
+[securebanking-auth-ui readme](securebanking-auth-ui/README.md)
```

Comment:
- Moved Auth app UI from https://github.com/OpenBankingToolkit/openbanking-common/tree/master/forgerock-openbanking-ui to Secure Banking UI project.
- Checked and build tested
- Depends currently of AM openbanking.
> This app can't work yet against Cloud platform.